### PR TITLE
Selection tool : colors fix

### PR DESCRIPTION
**Pull request by maxGimeno**
- This PR fixes a color problem with the selection tool, and some warnings about Signals/Slots. There is currently a segfault remaining when trying to create a points set from a selection, but it should be solved by another PR(#498) from @sgiraudot

### CODE REVIEW

File: Three/include/CGAL/Three/Scene_item.h [hunk 1]
```diff
@@ -55,6 +55,7 @@ class SCENE_ITEM_EXPORT Scene_item : public QObject {
 public:
   enum OpenGL_program_IDs { PROGRAM_WITH_LIGHT,
```
Comment (sloriot) [inline Three/include/CGAL/Three/Scene_item.h]: Is it expected that theses enum are duplicated with Viewer_interface.h?

**Comment by sloriot** [comment]
Merged in CGAL-4.8-Ic-56

**Comment by maxGimeno** [comment]
I think it is, ask @lrineau.

**Comment by lrineau** [comment]
About duplicating the enums... Actually I do not like that. I wrote it that way because I did not know, at that time, how to write a using declaration of enumerators from another class We should use:
```C++
using Viewer_interface::PROGRAM_WITH_LIGHT;
```
for each enumerator, because `using Viewer_interface::OpenGL_program_IDs` will only import the name of the enum, and not the enumerators.

**Comment by maxGimeno** [comment]
Ok, I will include that to the fix I am working on.

2015-11-24 12:56 GMT+01:00 Laurent Rineau <notifications@github.com>:

**Comment by sloriot** [comment]
Merged in CGAL-4.8-Ic-57

**Pull request merged**